qaframework-bdd-tests/src/test/resources/features/refapp-2.x/15-RolesAndPrivileges/AddUserAndAddRoleToUser.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Add User and Add Roles To User
+
+  Background: 
+    Given a user clicks on system administration app
+    Then the system loads manage users page
+
+  @selenium
+  @addUserAndAddRolesToUser
```

Comment:
@mherman22 what is your view if we replaced `@addUserAndAddRolesToUser `with `@user`?